src/app/app.module.ts

```diff
@@ -18,10 +18,24 @@ import { SpeakerDetailPage } from '../pages/speaker-detail/speaker-detail';
 import { SpeakerListPage } from '../pages/speaker-list/speaker-list';
 import { TabsPage } from '../pages/tabs/tabs';
 import { TutorialPage } from '../pages/tutorial/tutorial';
+import { Support } from '../pages/support/support';
 
 import { ConferenceData } from '../providers/conference-data';
 import { UserData } from '../providers/user-data';
 
+export const deepLinkConfig: DeepLinkConfig = {
+  links: [
+    { component: MapPage, name: 'map', segment: 'map' },
+    { component: SpeakerListPage, name: 'speakerList', segment: 'speaker-list' },
```

Comment:
Why is the name for `speakerList` camelCase when the others are not? Can these be consistent.